src/script/pages/app-testing.ts

```diff
@@ -0,0 +1,159 @@
+import {
+  LitElement,
+  css,
+  html,
+  customElement,
+  internalProperty,
+} from 'lit-element';
+import { Router } from '@vaadin/router';
+
+import { smallBreakPoint } from '../utils/breakpoints';
+
+import { runAllTests } from '../services/tests';
+import '../components/app-header';
+
+@customElement('app-testing')
+export class AppTesting extends LitElement {
+  @internalProperty() loading = false;
+  @internalProperty() currentPhrase: string = "PWABuilder is loading your PWA in the background";
+
+  static get styles() {
+    return css`
+      :host {
+        display: flex;
+        flex-direction: column;
+        background: url(/assets/images/background-copy.webp);
+        height: 100vh;
+        overflow: hidden;
+      }
+
+      #testing-container {
+        flex: 0.7;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+
+        text-align: center;
+
+        animation: 160ms fadeIn linear;
+      }
+
+      #testing-container img {
+        width: 466px;
+      }
+
+      #testing-container fast-progress {
+        position: fixed;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        margin-bottom: 0;
+      }
+
+      #testing-container fast-progress::part(progress) {
+        border-radius: 0;
+      }
+
+      #testing-container span {
+        font-weight: var(--font-bold);
+        font-size: var(--large-font-size);
+        margin-top: 12px;
+      }
+
+      @keyframes fadeIn {
+        from {
+          opacity: 0;
+        }
+
+        to {
+          opacity: 1;
+        }
+      }
```

Comment:
No change required, but a comment: this feels like something we could use in other places as well. If we did wish to reuse that, where is the place for it?